creation/interfaces-and-networks.md

```diff
@@ -733,3 +733,103 @@ spec:
 
 > **Note:** for some NICs (e.g. Mellanox), the kernel module needs to be
 > installed in the guest VM.
+
+### macvtap
+
+In `macvtap` mode, virtual machines are directly exposed to an host networking
+interface, and are connected directly to the L2 network connecting the hosts.
+The device is then passed through into the guest operating system as a char
+device.
+
+#### How to expose host interface to the macvtap device plugin
+To simplify the procedure, please use the
+[Cluster Network Addons Operator](https://github.com/kubevirt/cluster-network-addons-operator)
+to deploy and configure the macvtap components in your cluster. On how to use
+the operator, please refer to [their respective
+documentation](https://github.com/openshift/sriov-network-operator/blob/master/doc/quickstart.md).
+
+The aforementioned operator effectively deploys the
+[macvtap-cni](https://github.com/kubevirt/macvtap-cni) cni / device plugin
+combo.
+
+There are two different alternatives to configure which host interfaces get
+exposed to the user, enabling them to create macvtap interfaces on top of;
+  - select the host interfaces: indicates which host interfaces are exposed.
+  - expose all interfaces: all interfaces of all hosts are exposed.
+
+Both options are configured via the `macvtap-deviceplugin-config` ConfigMap,
```

Comment:
Would you show an example of the config map? After seeing it, it may be easier to connect the dots in the following NAD (why the heck do you use label "resourceName: eth0" and what is "macvtapnetwork"?